src/overlays/actors/ovl_En_Ending_Hero6/z_en_ending_hero6.c

```diff
@@ -9,9 +9,40 @@ void EnEndingHero6_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnEndingHero6_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnEndingHero6_Draw(Actor* thisx, GlobalContext* globalCtx);
 
+void func_80C23D60(EnEndingHero6* this, s32 npcIndex);
+void func_80C23DDC(EnEndingHero6* this);
 void func_80C23E18(EnEndingHero6* this, GlobalContext* globalCtx);
 
-#if 0
+extern FlexSkeletonHeader D_0600B0CC; // object_dt
+extern FlexSkeletonHeader D_06007908; // object_bai
+extern FlexSkeletonHeader D_06007150; // object_toryo
+extern FlexSkeletonHeader D_0600D640; // object_sdn
+extern FlexSkeletonHeader D_0600A850; // object_daiku
+
+extern AnimationHeader D_06000BE0; // object_dt
+extern AnimationHeader D_060011C0; // object_bai
+extern AnimationHeader D_06000E50; // object_toryo
+extern AnimationHeader D_06002A84; // object_sdn
+extern AnimationHeader D_06002FA0; // object_daiku
+
+// object_daiku
+extern Gfx D_060070C0[];
+extern Gfx D_06006FB0[];
+extern Gfx D_06006E80[];
+extern Gfx D_06006D70[];
+extern Gfx D_0600A390_dl[];
```

Comment:
```suggestion
extern Gfx D_0600A390[]; // display list
```
You can keep the name of the var to be the same as the segmented address until the proper object file is decompiled. Although if it is a display list, leaving a comment should be fine

Reply:
On this and the following comment: removing the suffix would result in a name clash with the declaration described in the following comment. This change prevents this. There is an earlier short discussion on this in the first set of comments on the PR (I do not know how to link comments, sorry).

Reply:
Ah interesting, yeah I found the convo, sorry I missed it. Sure, then yeah you can leave it as is!